sandbox/settings.py

```diff
@@ -105,6 +105,8 @@
 
 STATIC_URL = '/static/'
 
-TEMPLATE_DIRS = (os.path.join(BASE_DIR, 'accounts/templates'), )
+TEMPLATE_DIRS = (os.path.join(BASE_DIR, 'accounts/templates'),
+os.path.join(BASE_DIR, 'templates')
+)
```

Comment:
Aaaa, ok, no tak to jest niepotrzebne, ale to już było dodane wcześniej. Można to usunąć i zostawić sam wpis z templates. :+1: